LongReadRNABenchmark/lr_isoform_custom_docker/extract_denovo_model_stats.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import os
+import argparse
+
+def process_tracking(input_tracking_file, num_gtfs):
+	reliable_transcripts = 0
+	almost_reliable_transcripts = [0] * num_gtfs # supported by (num_gtfs - 2) tools
+	unique_transcripts = [0] * num_gtfs
+	missed_transcripts = [0] * num_gtfs
+	total_transcripts = [0] * num_gtfs
+
+	for line in open(input_tracking_file, "r"):
+		vals = line.strip().split()[4:]
+		assert len(vals) == num_gtfs
+
+		equality_vector = [0 if vals[i] == '-' else 1 for i in range(num_gtfs)]
+		for i, e in enumerate(equality_vector):
+			if e == 1:
+				total_transcripts[i] += 1
+
+		matches_transcripts = equality_vector.count(1)
+		assert matches_transcripts > 0
+
+		if matches_transcripts == num_gtfs:
+			reliable_transcripts += 1
+		elif matches_transcripts == num_gtfs - 1:
+			for i, e in enumerate(equality_vector):
+				if e == 1:
+					almost_reliable_transcripts[i] += 1
+			index = equality_vector.index(0)
+			missed_transcripts[index] += 1
+		elif matches_transcripts == 1:
+			index = equality_vector.index(1)
+			unique_transcripts[index] += 1
+
+	return reliable_transcripts, almost_reliable_transcripts, total_transcripts, unique_transcripts, missed_transcripts
+
+
+def print_denovo_stats(tool_name, reliable_transcripts, almost_reliable_transcripts, total_transcripts, unique_transcripts, missed_transcripts, out = sys.stdout):
+	out.write(tool_name + "\n")
```

Comment:
The data that comes out of this is very difficult to parse without knowing exactly how everything was run (and even then it's tricky).  I would propose outputting a csv table with columns of `reliable_transcripts, almost_reliable_transcripts, etc` and a row for each tool.  You can use pandas to handle it a bit easier as well I think.